src/primitive.h

```diff
@@ -70,6 +70,7 @@ namespace toit {
 #define MODULE_CORE(PRIMITIVE)               \
   PRIMITIVE(write_string_on_stdout, 2)       \
   PRIMITIVE(write_string_on_stderr, 2)       \
+  PRIMITIVE(get_std_handle, 1)               \
```

Comment:
Why the `get_` prefix? ... and you're not getting a handle back.

Maybe `std_file_descriptor'?